test.sh

```diff
@@ -1,35 +1,35 @@
 #!/usr/bin/env bash
 #
 # Run all etcd tests
-# ./test
-# ./test -v
+# ./test.sh
```

Comment:
Let me know if you want me to do in another PR? I just noticed when trying to run some unit tests.

Reply:
Feel free to smuggle it ;)